app_dart/dev/branch_regexps.txt

```diff
@@ -1,2 +1,4 @@
-^flutter-[0-9]+\.[0-9]+-candidate\.[0-9]+$
+^flutter-1+\.17+-candidate\.3+$
+^flutter-1+\.18+-candidate\.11+$
+^flutter-1+\.19+-candidate\.4+$
```

Comment:
This means we'll still call the github api instead of using the names here. I'd suggest to use directly:

flutter-1.17-candidate.3
flutter-1.18-candidate.11
flutter-1.19-candidate.4

And use the content of the file directly.

Reply:
Yes. You are right. It will still call the github api.

The main github traffic should fall on follow-up github prs/commits calls when iterating branch list.

The change here should reduce the github traffic a lot, but I will open another PR to remove the github call for the branch list.

Reply:
@keyonghan we don't need the `+`'s in these regexes, right?

Reply:
nvm, I see with your followup PR you're gonna remove this anyway.